lighthouse-core/driver/index.js

```diff
@@ -100,15 +100,19 @@ class Driver {
     const loadData = {};
     let pass = Promise.resolve();
 
+    if (config.traceName) {
+      loadData[config.traceName] = {};
```

Comment:
right, I meant dropping the `Object.assign` and assigning `tracingData[config.traceName] = loadData` directly